x-pack/plugins/security_solution/public/assistant/provider.tsx

```diff
@@ -36,6 +36,7 @@ export const AssistantProvider: React.FC = ({ children }) => {
   } = useKibana().services;
   const basePath = useBasePath();
   const isModelEvaluationEnabled = useIsExperimentalFeatureEnabled('assistantModelEvaluation');
+  const isStreamingEnabled = useIsExperimentalFeatureEnabled('aiStreamingEnabled');
```

Comment:
nit: consistent naming w/ other assistant FF's, i.e. `assistantStreamingEnabled`

No biggie though, I can format/update all the existing FF's when I work https://github.com/elastic/kibana/issues/172509

Reply:
done